lib/spec/openapi/utils.js

```diff
@@ -88,7 +88,7 @@ function normalizeUrl (url, serverUrls, stripBasePath) {
 
 function resolveServerUrls (servers) {
   const resolvedUrls = []
-  const findVariablesRegex = /{(.*?)}/g // As for OpenAPI v3 spec url variables are named in brackets, e.g. {foo}
+  const findVariablesRegex = /\{(.*?)\}/gu // As for OpenAPI v3 spec url variables are named in brackets, e.g. {foo}
```

Comment:
Was the escape necessary?

Just curious

Reply:
Without it they are treated as quantifiers.